firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/SessionReportingCoordinator.java

```diff
@@ -289,6 +300,19 @@ private CrashlyticsReport.Session.Event addLogsAndCustomKeysToEvent(
     return eventBuilder.build();
   }
 
+  private CrashlyticsReport.Session.Event addRolloutsStateToEvent(
+      CrashlyticsReport.Session.Event capturedEvent, UserMetadata reportMetadata) {
+    final CrashlyticsReport.Session.Event.Builder eventBuilder = capturedEvent.toBuilder();
```

Comment:
nit: do you think we should move this line inside the if block on line 308 since that's the only place it's being used?

Reply:
Ahhh, thanks for the suggestion, fixed!